pkg/cli/dump_test.go

```diff
@@ -858,3 +860,171 @@ INSERT INTO foo (id) VALUES
 		})
 	}
 }
+
+func TestDumpTempTables(t *testing.T) {
```

Comment:
I would just comment here why you didn't use the datadriven tests so that we remember